sharktuner/pyproject.toml

```
@@ -18,7 +22,21 @@ classifiers = [
 requires-python = ">= 3.10"
 
 # Version is set via the `setup.py`.
-dynamic = ["version"]
+dynamic = ["version", "dependencies", "optional-dependencies"]
 
 [project.urls]
 Repository = "https://github.com/nod-ai/shark-ai"
+
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["sharktuner*"]
```

```suggestion
include = ["sharktuner", "model_tuner"]
```
To package `model_tuner` this needs to be included here. Furthermore, I think we can drop the asterisk from `sharktuner`.